### DISCUSSION TITLE
customizable timing note coloring ( mania / synesthesia mod )

### DISCUSSION THREAD

**Discussion by christopher317** [ppy/osu#24036] [Ideas]
i don't think i need to do much more explaining
ideally there would be a preview as well but its not strictly necessary

default colors
![celluloid-shot0020](https://github.com/ppy/osu/assets/86526198/9f685f93-ddae-4f75-be99-a988fe62f2a9)
custom colors
![celluloid-shot0020](https://github.com/ppy/osu/assets/86526198/b4acea55-c033-40e6-8509-5b972aa655c2)
i think you get the idea.

( mainly for the sake of readability because some of the colors chosen have somewhat low contrast against a black background, pure red in particular )

**Comment by MegaGamer22** [comment]
please

**Comment by MegaGamer22** [comment]
As a Stepmania/ITG player, the colors confuse me a lot (4th notes are white rather than red and 8th notes are red rather than blue). Being able to customize these colors can improve readability while also preventing bad contrast with dark backgrounds.
